store/src/java-test/com/zimbra/cs/util/JWTBasedAuthTest.java

```diff
@@ -275,4 +276,30 @@ private UserServletContext mockServletConext(Account acct, boolean queryParamFlo
         UserServletContext context = new UserServletContext(req, resp, usrSrv);
         return context;
     }
+
+    @Test
+    public void testZimbraQoSFilterExtractUserId() {
+        HttpServletRequest req = EasyMock.createMock(HttpServletRequest.class);
+        Cookie cookies[] =  new Cookie[1];
+        String salt = "s1";
+        Cookie cookie = new Cookie("ZM_JWT", salt);
+        cookie.setHttpOnly(true);
+        cookies[0] = cookie;
+        try {
+            Account acct = Provisioning.getInstance().get(Key.AccountBy.name, "test@zimbra.com");
```

Comment:
Can we have rest specific email ids, I have seen that it causes conflicts when all tests are run.